src/app/ads/post/post.component.ts

```diff
@@ -1,15 +1,59 @@
 import { Component, OnInit } from '@angular/core';
+import { AdsService } from 'src/app/services/ads.service';
+import Ad from 'src/app/Models/ads.model';
+import User from 'src/app/Models/user.model';
+import { ActivatedRoute } from '@angular/router';
+import { UntypedFormBuilder } from '@angular/forms';
```

Comment:
what is this? remove it if it's unused.

Reply:
the Ad and ActivatedRoute modules are used
Ad is used in postInfo, 
ActivatedRoute is used in constructor

Reply:
I have deleted UntypedFormBuilder